starter-code/server.js

```diff
@@ -54,7 +57,7 @@ app.get('/articles', function(request, response) {
 
 app.post('/articles', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // Put your response here...3 this interacting with Article.prototype.insertRecord is update
```

Comment:
3, correct on the Article.prototype.insertRecord, but this is the CREATE part of CRUD